### ISSUE TITLE
libtiff 4.0.10 import error

### ISSUE THREAD

**Issue by jontwo** [AnacondaRecipes/pillow-feedstock#1]
Issue:
As discussed at https://github.com/scikit-image/scikit-image/issues/3679, there is a dll error with the latest version of `libtiff`, when importing from PIL.

```
λ python -c "import skimage;print skimage.__version__"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\ProgramData\Anaconda2\envs\farm\lib\site-packages\skimage\__init__.py", line 177, in <module>
    from .data import data_dir
  File "C:\ProgramData\Anaconda2\envs\farm\lib\site-packages\skimage\data\__init__.py", line 15, in <module>
    from ..io import imread, use_plugin
  File "C:\ProgramData\Anaconda2\envs\farm\lib\site-packages\skimage\io\__init__.py", line 7, in <module>
    from .manage_plugins import *
  File "C:\ProgramData\Anaconda2\envs\farm\lib\site-packages\skimage\io\manage_plugins.py", line 28, in <module>
    from .collection import imread_collection_wrapper
  File "C:\ProgramData\Anaconda2\envs\farm\lib\site-packages\skimage\io\collection.py", line 12, in <module>
    from PIL import Image
  File "C:\ProgramData\Anaconda2\envs\farm\lib\site-packages\PIL\Image.py", line 64, in <module>
    from . import _imaging as core
ImportError: DLL load failed: The specified module could not be found.
```
It works fine with `libtiff` 4.0.9.

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# Name                    Version                   Build  Channel
backports                 1.0                      py27_1
backports.functools_lru_cache 1.5                      py27_1
backports.shutil_get_terminal_size 1.0.0                    py27_2
backports_abc             0.5              py27h0ec6b72_0
blas                      1.0                         mkl
bokeh                     1.0.2                    py27_0
ca-certificates           2018.12.5                     0
certifi                   2018.11.29               py27_0
chardet                   3.0.4                    py27_1
click                     7.0                      py27_0
cloudpickle               0.6.1                    py27_0
colorama                  0.4.1                    py27_0
configobj                 5.0.6                    py27_1
cycler                    0.10.0           py27h59acbbf_0
cytoolz                   0.9.0.1          py27h0c8e037_1
dask                      1.0.0                    py27_0
dask-core                 1.0.0                    py27_0
decorator                 4.3.0                    py27_0
distributed               1.25.2                   py27_0
enum34                    1.1.6                    py27_1
farmlib                   0.28.9                    <pip>
farmlib                   0.28.13                  py27_0    file://<dev_server>/conda_packages_test
freetype                  2.9.1                h4d385ea_1
funcsigs                  1.0.2            py27h8885ae1_0
functools32               3.2.3.2                  py27_1
futures                   3.2.0                    py27_0
fuzzywuzzy                0.17.0                    <pip>
geos                      3.6.2                heec837b_2
h5py                      2.9.0            py27hb721d18_0
hdf5                      1.10.4               h530792d_0
heapdict                  1.0.0                    py27_2
icc_rt                    2019.0.0             h0cc432a_1
icu                       58.2                 h2aa20d9_1
imageio                   2.4.1                    py27_0
intel-openmp              2019.1                      144
ipython                   5.8.0                    py27_0
ipython_genutils          0.2.0            py27hbe997df_0
jbafarm                   0.28.11                   <pip>
jbagdal                   2.2.0                py27_vc9_0  [vc9]  file://<dev_server>/conda_packages
jbalitmus                 1.1.5                     <pip>
jinja2                    2.10                     py27_0
jpeg                      9b                   ha175dff_2
jsonschema                2.6.0            py27haaf3834_0
kiwisolver                1.0.1            py27hc56fc5f_0
krb5                      1.16.1               hb4d044e_6
libpng                    1.6.35               h7a46e7a_0
libpq                     10.5                 h1ac0bd9_0
libspatialindex           1.8.5                hc56fc5f_2
libtiff                   4.0.10            he490001_1001
linecache2                1.0.0                    py27_0
litmus                    1.1.5            py27h2f2e868_1    file://<dev_server>/conda_packages
locket                    0.2.0            py27h1ca288a_1
markupsafe                1.1.0            py27h0c8e037_0
matplotlib                2.2.3            py27h263d877_0
memory_profiler           0.55.0                   py27_0
mkl                       2018.0.3                      1
mkl_fft                   1.0.6            py27hac4a418_0
mock                      2.0.0            py27hc978a8d_0
msgpack-python            0.5.6            py27hdc96acc_1
networkx                  2.2                      py27_1
nose                      1.3.7                    py27_2
numpy                     1.15.4           py27hbe4291b_0
numpy-base                1.15.4           py27h2753ae9_0
olefile                   0.46                     py27_0
openssl                   1.0.2p               h0c8e037_0
packaging                 18.0                     py27_0
pandas                    0.22.0           py27hc56fc5f_0
partd                     0.3.9                    py27_0
pathlib2                  2.3.3                    py27_0
patsy                     0.5.1                    py27_0
pbr                       5.1.1                    py27_0
pickleshare               0.7.5                    py27_0
pillow                    5.3.0            py27h5b88493_0
pip                       18.1                     py27_0
proj4                     4.9.1                     vc9_0
prompt_toolkit            1.0.15           py27h3a8ec6a_0
psutil                    5.4.8            py27h0c8e037_0
psycopg2                  2.7.5            py27hc64555f_0
pygments                  2.3.1                    py27_0
pyparsing                 2.3.0                    py27_0
pyproj                    1.9.5.1                  py27_0
pyqt                      5.6.0            py27h6e61f57_6
pyreadline                2.1                      py27_1
python                    2.7.15               h2880e7c_4
python-dateutil           2.7.5                    py27_0
python-Levenshtein        0.12.0                    <pip>
pytz                      2018.7                   py27_0
pywavelets                1.0.1            py27hc997a72_0
pyyaml                    3.13             py27h0c8e037_0
qt                        5.6.2            vc9hc26998b_12
rtree                     0.8.3                    py27_0
scandir                   1.9.0            py27h0c8e037_0
scikit-image              0.14.1           py27hc56fc5f_0
scikit-learn              0.20.1           py27hc804a74_0
scipy                     0.19.1           py27h069396b_3
setuptools                40.6.3                   py27_0
shapely                   1.6.4            py27hc31c106_0
simplegeneric             0.8.1                    py27_2
singledispatch            3.4.0.3          py27h3f9d112_0
sip                       4.18.1           py27hc56fc5f_2
six                       1.12.0                   py27_0
sortedcontainers          2.1.0                    py27_0
sqlite                    3.22.0            vc9h9e24720_0  [vc9]  file://<dev_server>/conda_packages
statsmodels               0.9.0            py27hc997a72_0
tblib                     1.3.2            py27h8ae915c_0
tk                        8.6.8                h0c8e037_0
toolz                     0.9.0                    py27_0
tornado                   5.1.1            py27h0c8e037_0
tqdm                      4.29.1                     py_0
traceback2                1.4.0                    py27_0
traitlets                 4.3.2            py27h1b1b3a5_0
unittest2                 1.1.0                    py27_0
vc                        9                    h7299396_1
vs2008_runtime            9.00.30729.1         hfaea7d5_1
wcwidth                   0.1.7            py27hb1a0d82_0
wheel                     0.32.3                   py27_0
win_unicode_console       0.5              py27hc037021_0
wincertstore              0.2              py27hf04cefb_0
xlrd                      1.2.0                    py27_0
yaml                      0.1.7                h3e6d941_2
zict                      0.1.3                    py27_0
zlib                      1.2.11               h3cc03e0_3
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : farm
    active env location : C:\ProgramData\Anaconda2\envs\farm
            shell level : 1
       user config file : C:\Users\jontwo\.condarc
 populated config files : C:\Users\jontwo\.condarc
          conda version : 4.5.12
    conda-build version : 3.0.15
         python version : 2.7.13.final.0
       base environment : C:\ProgramData\Anaconda2  (writable)
           channel URLs : file://<dev_server>/conda_packages/win-64
                          file://<dev_server>/conda_packages/noarch
                          https://conda.anaconda.org/scitools/win-64
                          https://conda.anaconda.org/scitools/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/win-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/win-64
                          https://repo.anaconda.com/pkgs/pro/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\ProgramData\Anaconda2\pkgs
                          C:\Users\jontwo\AppData\Local\conda\conda\pkgs
       envs directories : C:\ProgramData\Anaconda2\envs
                          C:\Users\jontwo\AppData\Local\conda\conda\envs
                          C:\Users\jontwo\.conda\envs
               platform : win-64
             user-agent : conda/4.5.12 requests/2.14.2 CPython/2.7.13 Windows/10 Windows/10.0.17134
          administrator : False
             netrc file : None
           offline mode : False
```
</details>

**Comment by jjhelmus** [comment]
Thanks for reporting this. The `libtiff` 4.0.10 packages had some issues.  The troublesome packages have been revoked from the repository, new environments should pick up the working 4.0.9 package.  Corrected 4.0.10 packages should be available shortly.

**Comment by jjhelmus** [comment]
Corrected libtiff 4.0.10 packages are available in `pkgs/main`.

**Comment by jontwo** [comment]
Great. Thanks for the quick fix!
